### PR TITLE
Fix sentry-nativec build on windows

### PR DESCRIPTION
**Pull request by marandaneto**

### CODE REVIEW

File: sentry-android-ndk/build.gradle.kts [hunk 1]
```diff
@@ -38,6 +38,18 @@ android {
             cmake {
                 arguments.add(0, "-DANDROID_STL=c++_static")
                 arguments.add(0, "-DSENTRY_NATIVE_SRC=$sentryNativeSrc")
+
+                // https://android.googlesource.com/platform/ndk/+/master/docs/BuildSystemMaintainers.md#additional-required-arguments
+                arguments.add(0, "-DANDROID_PIE=ON")
+
+                // https://android.googlesource.com/platform/ndk/+/master/docs/BuildSystemMaintainers.md#os-versions
+                // do we need that if we have minSdkVersion?
```
Comment (DanAlbert) [inline sentry-android-ndk/build.gradle.kts]: Nope, this is automatic.

File: sentry-android-ndk/build.gradle.kts [hunk 2]
```diff
@@ -38,6 +38,18 @@ android {
             cmake {
                 arguments.add(0, "-DANDROID_STL=c++_static")
                 arguments.add(0, "-DSENTRY_NATIVE_SRC=$sentryNativeSrc")
+
+                // https://android.googlesource.com/platform/ndk/+/master/docs/BuildSystemMaintainers.md#additional-required-arguments
+                arguments.add(0, "-DANDROID_PIE=ON")
```
Comment (DanAlbert) [inline sentry-android-ndk/build.gradle.kts]: Shouldn't be necessary given the `minSdkVersion` of 18 used for native code. The CMake toolchain should configure this for you.

Comment (marandaneto) [reply sentry-android-ndk/build.gradle.kts]: thanks :)

File: sentry-android-ndk/build.gradle.kts (hunk 1 above)
Comment (marandaneto) [reply sentry-android-ndk/build.gradle.kts]: thanks

**Comment by marandaneto** [comment]
closing it, changes probably will be made on sentry-native

**Pull request closed without merging**